pie-website/src/components/Footer.js

```diff
@@ -39,11 +38,14 @@ function Footer() {
               </a>
             </div>
             <div className="sp">
-              101 O'Brien Hall, Berkeley, CA 94704 [
-              <a href="https://www.google.com/maps/d/u/0/edit?mid=1zVieVgGgfPfW8yv6wN9TDJZPzDOW3XL9&ll=37.87498246754354%2C-122.25991490933154&z=19">
-                map
+              <a href="https://www.google.com/maps/d/u/0/viewer?mid=1Sd6QurpSJK3xXEvOiISkWlKXj98&ll=37.87452100000002%2C-122.2592068&z=18">
+                101 O'Brien Hall, Berkeley, CA 94704
               </a>
-              ]
+            </div>
```

Comment:
Added this new div with the [LEAD center prescribed wording](https://lead.berkeley.edu/wp-content/uploads/2014/12/student-org-domain-guidelines.pdf)

```
RSOs must include the following language, in a visible and readable font, on every page of their site
(e.g. In the footer): We are a student group acting independently of the University of California. We
take full responsibility for our organization and this website.
```